intermediate-exercises/src/components/01-LifeCycle.js

```diff
@@ -1,44 +1,44 @@
 import React from 'react';
 
-/*
-  Tasks for this exercise are in comments inside the component class code.
-  In this exercise lifecycle methods will be used to provide convenient debug
-  messages in developer's console.
-  There are more lifecycle methods available.
-  Those three presented are commonly used.
- */
-
 class LifeCycle extends React.Component {
   constructor(props) {
     super(props);
     this.state = { name: 'Pesto' };
+    this.handleButtonClick = this.handleButtonClick.bind(this);
   }
 
-  // This code will be called when the component finishes mounting
-  // (so it is visible for a user).
   componentDidMount() {
-    // Task 1: Display a message "I'm mounted!" in developer's console when the
-    //         component finishes mounting.
-    //         Use `console.log` function for it.
+    console.log(`I'm mounted!`);
```

Comment:
you can either add eslint-no-console comment here
or try avoiding use of console.log